apps/ideaspace/src/components/common/IdeaForm/IdeaForm.js

```diff
@@ -7,227 +7,236 @@ import SubmissionButton from './SubmissionButton';
+                    id="features"
+                    name="features"
+                    touched={touched['features']}
+                    error={errors.features}
+                    cols={50}
+                    rows={5}
+                  />
+                </atoms.Box>
+                <Field
+                  as={organisms.OpenResponse}
+                  label="Anything else you would like to share to support your idea?:"
+                  placeholder="Want to share something else not listed above?"
+                  id="extraInfo"
+                  name="extraInfo"
+                  cols={50}
+                  rows={5}
+                />
+                <Field
+                  as={organisms.FormField}
+                  label="What's a catchy tagline for your idea that sums up your value and purpose: ?"
+                  placeholder="Your tagline"
+                  id="tagline"
+                  name="tagline"
+                />
 
-							<atoms.Box style={{ fontSize: '1rem', alignItems:'center'}}>
-								<Checkbox disabled={false} required/>
-								<atoms.Typography type='p'>
-									&nbsp;I have read and agree to the Terms and Conditions.<span style={{color:"red"}}>&nbsp;*</span>
-								</atoms.Typography>
-							</atoms.Box>
+                <atoms.Box flexDirection="column">
+                  <atoms.Typography
+                    type="label"
+                    style={{ marginLeft: '1rem', marginBottom: '0.5rem' }}
+                  >
+                    What level of involvement would you like to have after this
+                    submission?<span style={{ color: 'red' }}>&nbsp;*</span>
+                    <atoms.ToolTip
+                      content="As an “idea owner” you'll own the idea and be in charge of refine and update the information on the workshopping page."
+                      direction="top"
+                      delay={100}
+                      style={{ marginLeft: '0.1rem', marginBottom: '0.7rem' }}
+                    >
+                      <img alt="submit_image" src={popoverSvg} />
+                    </atoms.ToolTip>
+                  </atoms.Typography>
+                  <atoms.Box flexDirection="column" alignItems="flex-start">
+                    <atoms.Radio
+                      onChange={(e, label) =>
+                        setFieldValue('involveLevel', label)
+                      }
+                      label='I want to "own" this idea from beginning to end'
+                    />
+                    <atoms.Radio
+                      onChange={(e, label) =>
+                        setFieldValue('involveLevel', label)
+                      }
+                      label='I want to "own" this idea only after it is approved as a project'
+                    />
+                    <atoms.Radio
+                      onChange={(e, label) =>
+                        setFieldValue('involveLevel', label)
```

Comment:
Can we set field to "highly", so we don't need this conversion https://github.com/dev-launchers/dev-launchers-platform/pull/1944/files#r1744700300?

Reply:
@chungthuang yes thanks for the recommendation here, this is complete.